services/notification/notifiers/comment/conditions.py

```diff
@@ -207,6 +204,12 @@ async def check_condition_OR_group(
     async def check_condition(
         notifier: AbstractBaseNotifier, comparison: ComparisonProxy
     ) -> bool:
+        if comparison.pull and comparison.pull.commentid:
+            log.info(
```

Comment:
Should this be a warning?

Reply:
I don't think so... It's the expected behavior after all.
I'm adding this just to help us with debugging in the future